client/components/UserDecalPanel.js

```diff
@@ -54,13 +53,36 @@ class UserDetailsPanel extends Component {
     })
   }
 
-  static _renderClaimedAtRow (row) {
-    const rescue = row.original
+  _handleToggle = (event) => {
```

Comment:
I'd name this something more descriptive like
```suggestion
  _handleDecalVisibilityToggle = (event) => {
```